### ISSUE TITLE
cannot build app

### ISSUE THREAD

**Issue by pranavpandey1998official** [wtg/Flutter_ShuttleTracker#59]
/Users/pranav/.pub-cache/hosted/pub.dartlang.org/sqflite-1.2.1/ios/Classes/SqflitePlugin.m:2:9: warning: non-portable path to file '<fmdb/FMDB.h>'; specified path differs in case from file name
    on disk [-Wnonportable-include-path]
    #import <FMDB/FMDB.h>
            ^~~~~~~~~~~~~
            <fmdb/FMDB.h>
    1 warning generated.

    Compiler message:
    ../../../.pub-cache/hosted/pub.dartlang.org/flutter_platform_widgets-0.41.0/lib/src/platform_app.dart:66:23: Error: Type 'ActionFactory' not found.
      final Map<LocalKey, ActionFactory> actions;
                          ^^^^^^^^^^^^^
    ../../../.pub-cache/hosted/pub.dartlang.org/flutter_platform_widgets-0.41.0/lib/src/platform_app.dart:95:21: Error: Type 'ActionFactory' not found.
          Map<LocalKey, ActionFactory> actions,
                        ^^^^^^^^^^^^^
    ../../../.pub-cache/hosted/pub.dartlang.org/flutter_platform_widgets-0.41.0/lib/src/platform_app.dart:151:21: Error: Type 'ActionFactory' not found.
          Map<LocalKey, ActionFactory> actions,
                        ^^^^^^^^^^^^^
    ../../../.pub-cache/hosted/pub.dartlang.org/flutter_platform_widgets-0.41.0/lib/src/platform_app.dart:218:23: Error: Type 'ActionFactory' not found.
      final Map<LocalKey, ActionFactory> actions;
                          ^^^^^^^^^^^^^
    ../../../.pub-cache/hosted/pub.dartlang.org/flutter_platform_widgets-0.41.0/lib/src/platform_app.dart:66:23: Error: 'ActionFactory' isn't a type.
      final Map<LocalKey, ActionFactory> actions;
                          ^^^^^^^^^^^^^
    ../../../.pub-cache/hosted/pub.dartlang.org/flutter_platform_widgets-0.41.0/lib/src/platform_app.dart:95:21: Error: 'ActionFactory' isn't a type.
          Map<LocalKey, ActionFactory> actions,
                        ^^^^^^^^^^^^^
    ../../../.pub-cache/hosted/pub.dartlang.org/flutter_platform_widgets-0.41.0/lib/src/platform_app.dart:151:21: Error: 'ActionFactory' isn't a type.
          Map<LocalKey, ActionFactory> actions,
                        ^^^^^^^^^^^^^
    ../../../.pub-cache/hosted/pub.dartlang.org/flutter_platform_widgets-0.41.0/lib/src/platform_app.dart:218:23: Error: 'ActionFactory' isn't a type.
      final Map<LocalKey, ActionFactory> actions;
                          ^^^^^^^^^^^^^
    ../../../.pub-cache/hosted/pub.dartlang.org/flutter_platform_widgets-0.41.0/lib/src/platform_app.dart:306:30: Error: The argument type 'Map<LocalKey, invalid-type>' can't be assigned to the
    parameter type 'Map<Type, Action<Intent>>'.
     - 'Map' is from 'dart:core'.
     - 'LocalKey' is from 'package:flutter/src/foundation/key.dart' ('../../../flutter/packages/flutter/lib/src/foundation/key.dart').
     - 'Type' is from 'dart:core'.
     - 'Action' is from 'package:flutter/src/widgets/actions.dart' ('../../../flutter/packages/flutter/lib/src/widgets/actions.dart').
     - 'Intent' is from 'package:flutter/src/widgets/actions.dart' ('../../../flutter/packages/flutter/lib/src/widgets/actions.dart').
          actions: data?.actions ?? actions,
                                 ^
    ../../../.pub-cache/hosted/pub.dartlang.org/flutter_platform_widgets-0.41.0/lib/src/platform_app.dart:359:30: Error: The argument type 'Map<LocalKey, invalid-type>' can't be assigned to the
    parameter type 'Map<Type, Action<Intent>>'.
     - 'Map' is from 'dart:core'.
     - 'LocalKey' is from 'package:flutter/src/foundation/key.dart' ('../../../flutter/packages/flutter/lib/src/foundation/key.dart').
     - 'Type' is from 'dart:core'.
     - 'Action' is from 'package:flutter/src/widgets/actions.dart' ('../../../flutter/packages/flutter/lib/src/widgets/actions.dart').
     - 'Intent' is from 'package:flutter/src/widgets/actions.dart' ('../../../flutter/packages/flutter/lib/src/widgets/actions.dart').
          actions: data?.actions ?? actions,
                                 ^
    ../../../.pub-cache/hosted/pub.dartlang.org/flutter_cupertino_settings-0.3.0/lib/widgets/secret.dart:41:58: Error: Getter not found: 'obscuringCharacter'.
                      _show ? widget.secret : RenderEditable.obscuringCharacter * widget.secret.length,
                                                             ^^^^^^^^^^^^^^^^^^
    Target kernel_snapshot failed: Exception: Errors during snapshot creation: null
    build failed.
    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Planning build
    note: Constructing build description

**Comment by samuelobe** [comment]
Hello thank you for submitting an issue! First of, are you in the stable channel of Flutter and have you updated Flutter to the newest version? I ran into a error like this when I tried to run the app in the master and dev channels. 

Update: The dependency that is causing most of the errors, flutter_platform_widgets, has limited support for non-stable channels. I would assume the same is true for the other error causing dependency.
